maps/clarion.dmm

```diff
@@ -27664,15 +27664,16 @@
 "aYe" = (
 /obj/machinery/light_switch/north,
 /obj/table/wood/auto,
+/obj/machinery/firealarm{
```

Comment:
no changes are permitted to clarion at this time

if you wish to make changes you must ask dions